addon/href-to.js

```diff
@@ -26,9 +26,10 @@ export default class {
 
   handle() {
     let router = this._getRouter();
+    let urlWithoutRoot = this.getUrlWithRoot();
```

Comment:
Is this withRoot or withoutRoot? variable says one thing method name says another